admin-dev/themes/new-theme/js/components/translatable-input.js

```diff
@@ -26,8 +26,15 @@
 const $ = window.$;
 
 class TranslatableInput {
-    constructor() {
-        $('body').on('click', '.js-locale-item', this.toggleInputs);
+    constructor(options) {
+        options = options || {};
+
+        const self = this;
+        self.localeItemSelector = options.localeItemSelector || '.js-locale-item';
```

Comment:
why do you need `self` variable instead of directly using `this`?

Reply:
no reason, you're right ^^